packages/fast-colors/app/index.tsx

```diff
@@ -0,0 +1,13 @@
+import * as React from "react";
```

Comment:
Since this is going into the v-next branch, I believe we're updating all instances of `import * as ____` to `import ____ from "";` as we are leveraging esModuleInterop

Reply:
Those should all be updated now